webapp/src/pages/EditUserPage/EditUserPage.tsx

```diff
@@ -18,14 +80,48 @@ export default function EditUserPage() {
         XCoordinate: authUser!.avatarX,
         YCoordinate: authUser!.avatarY,
       }}
+      secondaryButton={{
+        children: 'save',
+        variant: ButtonVariant.SUBMIT,
+        form: 'edit-user-form',
+      }}
+      button={{
+        children: 'edit password',
+        variant: ButtonVariant.ALTERNATIVE,
+        onClick: () => navigate(EDIT_USER_PASSWORD_URL),
+      }}
```

Comment:
thought: The users who log in with OAuth have a `null` password. Should we allow them to assign a password so they can also log in with a username + password?
Or do we want to hide this button for OAuth users?

To check if a user is local (username + password) or from an OAuth provider, we could add a new field to the user, `isLocal`?, which we serialize and assign to true if the user doesn't have a password or false if they do, similar to what we have for the chatroom (`isPublic`).

Maybe we could leave a TODO until we can test this.

Reply:
You are right. We can do something like that, sounds good. I will leave a TODO and create and issue